mlir/utils/tuna/tuna-script.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# For installing mysql 8.0 for testing, or for running with an isolated database.
+function mysql_setup_generic
+{
+    # Latest "generic" version.  (Must have libaio1 installed already.)
+    # Note that all this happens without privileges.
```

Comment:
rocMLIR CI jobs run as a normal user.  Embedding mysqld in our docker image is (a) large and (b) painful.  The pain comes from not having systemd in docker-world and then needing root to start things up.  Instead, we install the "generic" mysql, which doesn't care about systemd, and then arrange to run it as a normal user, in part by using a socket instead of dealing with network ports and authorisation.